mod/health.php

```diff
@@ -1,143 +1,138 @@
+					'<span class="health ' . health_score_to_name($site['health_score']) . '">&hearts;</span> ' .
+					'<a href="/health/' . $site['id'] . '">' . $site['base_url'] . '</a> ' .
+					'(' . $site['users'] . ')' .
+					($site['effective_base_url'] ? ' -&gt; <abbr title="Redirects to this domain.">' . $site['effective_base_url'] . '</abbr>' : '') .
 					"<br />\r\n";
 			}
-			
-			
-		}
-		
-		else {
+		} else {
 			$result = 'No results';
 		}
-		
 	}
-	
+
 	$tpl .= file_get_contents('view/health_search.tpl');
+
 	return replace_macros($tpl, array(
 		'$searched' => $search,
 		'$result' => $result
 	));
-	
 }
 
-function health_summary(&$a){
-	
+function health_summary(App $a)
+{
 	$sites = array();
-	
+
 	//Find the user count per site.
 	$r = q("SELECT `homepage` FROM `profile`");
-	if(count($r)) {
-		foreach($r as $rr) {
+	if (count($r)) {
+		foreach ($r as $rr) {
 			$site = parse_site_from_url($rr['homepage']);
-			if($site) {
-				if(!isset($sites[$site]))
-					$sites[$site] = 0;		
+			if ($site) {
+				if (!isset($sites[$site])) {
+					$sites[$site] = 0;
+				}
 				$sites[$site] ++;
 			}
 		}
 	}
-	
+
 	//See if we have a health for them AND they provide SSL.
 	$sites_with_health = array();
 	$site_healths = array();
-	
+
 	$r = q("SELECT * FROM `site-health` WHERE `reg_policy`='REGISTER_OPEN' AND `ssl_state` = 1");
-	if(count($r)) {
-		foreach($r as $rr) {
-			$sites_with_health[$rr['base_url']] = (($sites[$rr['base_url']] / 100) + 10) * intval($rr['health_score']);
-			$site_healths[$rr['base_url']] = $rr;
+	if (count($r)) {
+		foreach ($r as $rr) {
+			if (isset($sites[$rr['base_url']])) {
+				$sites_with_health[$rr['base_url']] = (($sites[$rr['base_url']] / 100) + 10) * intval($rr['health_score']);
+				$site_healths[$rr['base_url']] = $rr;
+			}
 		}
 	}
-	
+
 	arsort($sites_with_health);
 	$total = 0;
 	$public_sites = '';
-	foreach($sites_with_health as $k => $v)
-	{
-		
+	foreach ($sites_with_health as $k => $v) {
 		//Stop at unhealthy sites.
 		$site = $site_healths[$k];
-		if($site['health_score'] <= 20) break;
-		
+		if ($site['health_score'] <= 20) {
+			break;
+		}
+
 		//Skip small sites.
 		$users = $sites[$k];
-		if($users < 5) continue;
-		
+		if ($users < 5) {
+			continue;
+		}
+
 		$public_sites .=
-			'<span class="health '.health_score_to_name($site['health_score']).'">&hearts;</span> '.
-			'<a href="/health/'.$site['id'].'">' . $k . '</a> '.
-			'(' . $users . ')'.
+			'<span class="health ' . health_score_to_name($site['health_score']) . '">&hearts;</span> ' .
+			'<a href="/health/' . $site['id'] . '">' . $k . '</a> ' .
+			'(' . $users . ')' .
 			"<br />\r\n";
 		$total ++;
-		
 	}
 	$public_sites .= "<br>Total: $total<br />\r\n";
-	
-	$tpl .= file_get_contents('view/health_summary.tpl');
+
+	$tpl = file_get_contents('view/health_summary.tpl');
+
 	return replace_macros($tpl, array(
-		'$versions' => $versions,
```

Comment:
I have no idea, the variable didn't exist in the controller nor in the template.